### PR TITLE
feat: Add option to disable the window frame

### PR DESCRIPTION
**Pull request by zt64**

### CODE REVIEW

**Comment by AAGaming00** [comment]
guhhh useless almost all wm can do this

insane

File: src/patcher.ts [hunk 1]
```diff
@@ -65,12 +65,15 @@ if (!process.argv.includes("--vanilla")) {
         };
     }
 
+    const settings = JSON.parse(readSettings());
```
Comment (Vendicated) [inline src/patcher.ts:68]: should be in try catch in case user corrupts settings, like from where u removed this

Comment (zt64) [reply src/patcher.ts:68]: Should I remove the other try catch or keep that too

Comment (zt64) [reply src/patcher.ts]: I left it

File: src/patcher.ts [hunk 2]
```diff
@@ -118,7 +124,6 @@ if (!process.argv.includes("--vanilla")) {
         });
 
         try {
-            const settings = JSON.parse(readSettings());
             if (settings.enableReactDevtools)
```
Comment (Vendicated) [inline src/patcher.ts:127]:
```suggestion
            if (settings?.enableReactDevtools)
```

**Pull request merged**